Sources/Identity.swift

```diff
@@ -42,15 +43,58 @@ import Foundation
     }
 
     public func readyForEvent(_ event: Event) -> Bool {
+        if event.urlVariables, getSharedState(extensionName: IdentityConstants.SharedState.Configuration.SHARED_OWNER_NAME, event: event)?.value == nil { return false}
+
         return state.bootupIfReady(getSharedState: getSharedState(extensionName:event:),
                                    createXDMSharedState: createXDMSharedState(data:event:))
     }
 
     // MARK: Event Listeners
 
-    /// Handles events requesting for identifiers. Dispatches response event containing the identifiers. Called by listener registered with event hub.
+    /// Handles events requesting for identifiers. Called by listener registered with event hub.
     /// - Parameter event: the identity request event
     private func handleIdentityRequest(event: Event) {
+        if event.urlVariables {
+            processGetUrlVariablesRequest(event: event)
+        } else {
+            processGetIdentifiersRequest(event: event)
+        }
+    }
+
+    /// Handles events requesting for url variables. Dispatches response event containing the url variables string.
+    /// - Parameter event: the identity request event
+    func processGetUrlVariablesRequest(event: Event) {
+        guard let configurationSharedState = getSharedState(extensionName: IdentityConstants.SharedState.Configuration.SHARED_OWNER_NAME, event: event)?.value else {
+            Log.warning(label: friendlyName, "\(#function) - Cannot process getUrlVariables request Identity event, configuration not found.")
+            return
+        }
+
+        // if config doesn't have org id we cannot proceed.
```

Comment:
nit: it is preferred to use impersonal sentences in docs or comments, in this case instead of using "if config doesn't have org id **we cannot proceed.**" it would be something like: "org is required to process the URL variables request"